components/SignUpLayout/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import SignUpSidebar from "./SignUpSidebar";
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const SignUpLayout: React.FC<Props> = ({ children }: Props) => {
+  return (
+    <div>
+      <SignUpSidebar />
```

Comment:
are you not able to just put `<div className="fixed top-0 flex flex-col justify-between w-64 h-screen bg-blue-muted" />` here or does that make the display buggy?

Reply:
oh yes fixed!